templates/layout/_form_media.html.twig

```diff
@@ -3,7 +3,7 @@
         <img class="image-preview" src="#">
         <button type="button" class="image-preview-clear" data-action="image-upload#clearPreview">x</button>
     </div>
-    <div>
+    <div style="width:300px;">
```

Comment:
Inline style is not recommend anymore, try to add a class. And use CSS.

Reply:
Done